pkg/runfileconfig/run_file_config_parser.go

```diff
@@ -97,6 +97,16 @@ func (a *RunFileConfig) validateRunConfig(runFilePath string) error {
 		if len(strings.TrimSpace(a.Apps[i].ResourcesPath)) > 0 {
 			a.Apps[i].ResourcesPaths = append(a.Apps[i].ResourcesPaths, a.Apps[i].ResourcesPath)
 		}
+
+		// Check containerImagePullPolicy is valid
+		if a.Apps[i].ContainerImagePullPolicy != "" {
+			allowedValues := []string{"Always", "Never", "IfNotPresent"}
```

Comment:
nit: It might be better to move up as `const` to avoid instantiation on each app and give a meaningful name.